src/h3/_cy/cells.pyx

```diff
@@ -164,10 +164,10 @@ cpdef H3int[:] children(H3int h, res=None):
 
     return mv
 
-cpdef H3int center_child(H3int h, res=None) except 0:
+cpdef H3int center_child(H3int h, int res=-1) except 0:
```

Comment:
A note for future discussion: Presumably typing every input argument would lead to faster cython code, so that the generated C code doesn't need to do a ton of type checking. So defining this arg as `int res=-1` would (I'm guessing) be faster than `res=None`.

But I'll revert this for now so that this PR has minimal changes